.gitignore

```diff
@@ -24,3 +24,6 @@ profiles.clj
 resources/contracts
 node_modules
 .DS_Store
+src/java
+yarn.lock
```

Comment:
As far as I know we're not using yarn, are we?

Reply:
right, we're not, will drop that.